src/Pact/Eval.hs

```diff
@@ -1278,13 +1350,15 @@ applyPact i app (TList steps _ _) PactStep {..} = do
         || (not private && _psRollback) -- done if public rollback
         || (private && _psRollback && _psStep == 0) -- done if private and rolled back to step 0
 
+  when (any (\npe -> (_npeStep npe /= _psStep) && (_npeStep npe < (_npeStepCount npe - 1))) (_peNested resultState)) $
```

Comment:
make a function of this to use here and with L1290

Reply:
:+1: